src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs

```diff
@@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
                 jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
```

Comment:
A chance at some point but there will have to be SIGNIFICANT demand in our
customer base, and well there is not currently ;-)
On Tue, May 24, 2016 at 10:32 PM Matthew Erbs <notifications@github.com>
wrote:

> In src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs
> <https://github.com/serilog/serilog-sinks-splunk/pull/15#discussion_r64517345>
> :
>
> > @@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
> >                  jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
>
> We are using ConcurrentQueue. I understand it has a some overhead.
> @glennblock <https://github.com/glennblock>, any chance you guys are
> going to port to dotnet core?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/serilog/serilog-sinks-splunk/pull/15/files/c965fe65ed3789486169001735cf5b68fa135c6d#r64517345>
>